src/main/java/uk/gov/di/ipv/cri/common/library/domain/personidentity/PersonIdentity.java

```diff
@@ -7,6 +7,8 @@
 import java.util.List;
 
 public class PersonIdentity {
+
+    private String personalNumber;
```

Comment:
Should this not be inside SocialSecurityRecord to avoid getting confused with document details